webui/react/src/hooks/useModal/Project/useModalProjectCreate.tsx

```diff
@@ -20,12 +21,22 @@ interface Props {
 const useModalProjectCreate = ({ onClose, workspaceId }: Props): ModalHooks => {
   const [ name, setName ] = useState('');
   const [ description, setDescription ] = useState('');
+  const [ isNameUnique, setIsNameUnique ] = useState<boolean>(true);
 
   const { modalOpen: openOrUpdate, modalRef, ...modalHook } = useModal({ onClose });
 
+  const findIsNameUnique = useCallback(async (name: string) => {
+    const projectList = await getWorkspaceProjects({ id: workspaceId });
```

Comment:
**chore:** API calls, we'll want to wrap in a try/catch block because it can cause errors to be thrown.

Reply:
just noting that this won't be needed if we let the backend handle the not-a-unique-name catching

Reply:
Added better error message  as a replacement of this https://github.com/determined-ai/determined/pull/4691#discussion_r936965850